lib/assets/summary.html.erb

```diff
@@ -189,6 +189,11 @@
 
               <% last_size = screenshot.screen_size %>
             <% end %>
+            
+            <% if last_size != nil %>
+              </div>
```

Comment:
Which `div` is closed here? Is it `<div class="app-screenshot-row">`? If so, shouldn't this be inside the `elsif` block` right after assigning `last_size` instead?